Gemfile

```diff
@@ -2,3 +2,9 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in stack_master.gemspec
 gemspec
+
+if RUBY_VERSION >= '3.0.0'
+  # SparkleFormation has an issue with Ruby 3 and the SortedSet class.
+  # Remove after merged: https://github.com/sparkleformation/sparkle_formation/pull/271
+  gem 'faux_sorted_set', require: false
+end
```

Comment:
SparkleFormation has a problem running on Ruby 3. It uses meta-programming to walk the object graph, and an error is raised when it visits the `SortedSet` constant. The class was removed from the Ruby standard library in version 3.0. Now the standard library raises an informative error if the `SortedSet` constant is referenced when an implementation has not been provided via a gem.

Remove this work-around once a fix is merged and released. Perhaps https://github.com/sparkleformation/sparkle_formation/pull/271